packages/experimental/Expander/src/Expander.tsx

```diff
@@ -24,27 +24,24 @@ export const Expander = compose<ExpanderType>({
     const expanderHeight = expandedState? userProps.expandedHeight : userProps.collapsedHeight;
 
     const _onCollapsing = async () => {
-      if (userProps.onCollapsing !== undefined) {
-        userProps.onCollapsing();
-      }
+      userProps.onCollapsing && userProps.onCollapsing();
```

Comment:
I think you can write this as `userProps?.onCollapsing()`

Reply:
This still calls the function even if the user didn't provide a callback, and that throws an error

Reply:
I put the optional chaining on the wrong thing :D :
`userProps.onCollapsing?.()` this will call only if onCollapsing is not falsy

Reply:
TIL I have learned the interesting syntax of `?.()` ...